### PR TITLE
[gosnmp] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
@SuperQ @TimRots does this look good?

### CODE REVIEW

File: projects/gosnmp/project.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/gosnmp/gosnmp"
+language: go
+primary_contact: "superq@gmail.com"
+auto_ccs:
+  - "ajsinghyadav00@gmail.com"
```
Comment (TimRots) [inline projects/gosnmp/project.yaml:5]: Bit confused why you slipstream yourself into the CC versus project maintainers?

Comment (0x34d) [reply projects/gosnmp/project.yaml:5]: [primary_contact, auto_ccs](https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary)  

[primary_contact](https://github.com/search?q=repo%3Agoogle%2Foss-fuzz+primary_contact+language%3AYAML&type=code&l=YAML) is usually the main maintainer of the project and `auto_ccs`  everyone Else.

Comment (jonathanmetzman) [reply projects/gosnmp/project.yaml:6]: It's good to have the contributor of an integration CCed because they are most able to fix it when it is broken.

**Comment by jonathanmetzman** [comment]
@SuperQ @TimRots do you approve?

**Review by SuperQ — APPROVED** [review]
LGTM

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**